pact-lang-api.js

```diff
@@ -52,7 +52,7 @@ var hash = function(s) {
  */
 var base64UrlEncode = function(s) {
   return base64url.escape(base64url.encode(s));
-}
+};
```

Comment:
do we need these or not? asking b/c on line 213 below it looks like there isn't a semicolon. We should standardize and do all or none, your call.

Reply:
Removed all semicolons from the function closure brackets.